docs/contracts/contracts/LSP0ERC725Account/LSP0ERC725Account.md

```diff
@@ -0,0 +1,1485 @@
+- When notifying the new owner via LSP1, the typeId used MUST be the `keccak256(...)` hash of [LSP0OwnershipTransferred_RecipientNotification].
+
+</blockquote>
+
+### batchCalls
+
+:::note Links
+
+- Specification details in [**LSP-0-ERC725Account**](https://github.com/lukso-network/lips/tree/main/LSPs/LSP-0-ERC725Account.md#batchcalls)
+- Solidity implementation in [**LSP0ERC725Account**](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725Account.sol)
+- Function signature: `batchCalls(bytes[])`
+- Function selector: `0x6963d438`
+
+:::
+
+```solidity
+function batchCalls(bytes[] data) external nonpayable returns (bytes[] results);
+```
+
+Allows a caller to batch different function calls in one call. Perform a `delegatecall` on self, to call different functions with preserving the context It is not possible to send value along the functions call due to the use of `delegatecall`.
+
+#### Parameters
+
+| Name   |   Type    | Description                                                          |
+| ------ | :-------: | -------------------------------------------------------------------- |
+| `data` | `bytes[]` | An array of ABI encoded function calls to be called on the contract. |
+
+#### Returns
+
+| Name      |   Type    | Description                                            |
+| --------- | :-------: | ------------------------------------------------------ |
+| `results` | `bytes[]` | An array of values returned by the executed functions. |
+
+### execute
+
+:::note Links
+
+- Specification details in [**LSP-0-ERC725Account**](https://github.com/lukso-network/lips/tree/main/LSPs/LSP-0-ERC725Account.md#execute)
+- Solidity implementation in [**LSP0ERC725Account**](https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/LSP0ERC725Account/LSP0ERC725Account.sol)
+- Function signature: `execute(uint256,address,uint256,bytes)`
+- Function selector: `0x44c028fe`
+
+:::
+
+```solidity
+function execute(
+  uint256 operationType,
+  address target,
+  uint256 value,
+  bytes data
+) external payable returns (bytes);
+```
+
+_Calling address `target` using `operationType`, transferring `value` wei and data: `data`. _
+
+Generic executor function to:
+
+- send native tokens to any address.
+
+- interact with any contract by passing an abi-encoded function call in the `data` parameter.
+
+- deploy a contract by providing its creation bytecode in the `data` parameter.
+
+<blockquote>
+
+**Requirements:**
+
+- If a `value` is provided, the contract MUST have at least this amount in its balance to execute successfully.
+- If the operation type is `STATICCALL` or `DELEGATECALL`, `value` SHOULD be 0.
+- `target` SHOULD be `address(0)` when deploying a contract.
+- MUST pass when called by the owner or by an authorised address that pass the verification check performed on the owner accordinng to [LSP-20-CallVerification] specification.
```

Comment:
This whole sentence needs re-wording.
We cannot write an API/ABI specs the same way as we write a formal specification document.

- LSP specs = how it should work / intended to work (for "designers, architects")
- ABI Technical docs = how it actually works, how the software and its functionalities can be used.